spikeextractors/baseextractor.py

```diff
@@ -67,9 +67,9 @@ def dump_to_json(self, file_path=None):
         '''
         if self.check_if_dumpable():
             if file_path is None:
-                if 'Recording' in self.__class__:
+                if 'Recording' in str(self.__class__):
```

Comment:
or self.__class__.__name__

Reply:
that doesn't work

Reply:
`type(self).__name__` works as well, or `str(self.__class__)`

Reply:
same as thos type(self).__name__